.azure-pipelines/win32-conda-CI.yml

```diff
@@ -167,10 +181,25 @@ jobs:
   - script: |
       call activate skl2onnxEnvironment
       if "$(onnx.target_opset)" neq "" set TEST_TARGET_OPSET=$(onnx.target_opset)
-      pip install lightgbm xgboost
+      pip install lightgbm xgboost pydot
       pip install onnxmltools --no-deps
     displayName: 'install onnxmltools'
 
+  - script: |
+      call activate skl2onnxEnvironment
+      if "$(onnx.target_opset)" neq "" set TEST_TARGET_OPSET=$(onnx.target_opset)
+      python -m pip install git+https://github.com/xadupre/onnxmltools.git@i396xgb
```

Comment:
Why is there a reference to your forked branch?

Reply:
Because this one has not yet been merged and is needed.

Reply:
Shoudn't we first merge the necessary changes then? Not sure if it is a good idea to reference a forked branch.

Reply:
You're right. That's my plan. I'm waiting the PR https://github.com/onnx/onnxmltools/pull/397 to be merged.